api/src/services/organizations_v1/create_organization_invitation.py

```diff
@@ -20,6 +23,55 @@
 INVITATION_EXPIRY_DAYS = 7
 
 
+def _send_invitation_email(
+    invitation: OrganizationInvitation,
+    organization,
+    invitee_email: str,
+) -> None:
+    """Send invitation email to the invitee.
+
+    This function handles all email sending logic including configuration,
+    error handling, and logging. Email failures do not raise exceptions.
+
+    Args:
+        invitation: The created invitation record
+        organization: The organization the user is being invited to
+        inviter: The user who created the invitation
+        invitee_email: Email address of the invitee
+    """
+    try:
```

Comment:
A little nitpicky, rather than wrapping everything here in a try block, could we just wrap the email send which is the only part I think we would meaningfully expect a failure.